ext/Pod-Html/lib/Pod/Html.pm

```diff
@@ -2,20 +2,27 @@ package Pod::Html;
 use strict;
 use Exporter 'import';
 
-our $VERSION = 1.28;
-our @EXPORT = qw(pod2html htmlify);
-our @EXPORT_OK = qw(anchorify relativize_url);
+our $VERSION = 1.29;
+eval $VERSION;
```

Comment:
Preferred mechanism for this: `$VERSION =~ tr/_//d;`

Reply:
> Preferred mechanism for this: `$VERSION =~ tr/_//d;`

That may be your preference, but it is evidently not the preference of people who up until now have contributed to the core distribution.
```
$ ack -l 'eval \$VERSION' cpan dist ext lib |sort
cpan/Compress-Raw-Bzip2/lib/Compress/Raw/Bzip2.pm
cpan/Compress-Raw-Zlib/lib/Compress/Raw/Zlib.pm
cpan/DB_File/DB_File.pm
cpan/ExtUtils-Install/lib/ExtUtils/Install.pm
cpan/ExtUtils-Install/lib/ExtUtils/Installed.pm
cpan/ExtUtils-Install/lib/ExtUtils/Packlist.pm
cpan/File-Fetch/lib/File/Fetch.pm
cpan/File-Path/lib/File/Path.pm
cpan/Filter-Util-Call/Call.pm
cpan/IO-Compress/lib/Compress/Zlib.pm
cpan/Module-Metadata/t/extract-version.t
cpan/Win32/Win32.pm
dist/ExtUtils-ParseXS/lib/ExtUtils/ParseXS.pm
dist/Thread-Queue/lib/Thread/Queue.pm
dist/Thread-Semaphore/lib/Thread/Semaphore.pm
dist/Time-HiRes/HiRes.pm
dist/threads-shared/lib/threads/shared.pm
dist/threads/lib/threads.pm
ext/Devel-Peek/Peek.pm
ext/Errno/Errno.pm
lib/Compress/Raw/Bzip2.pm
lib/Compress/Raw/Zlib.pm
lib/Compress/Zlib.pm
lib/DB_File.pm
lib/Devel/Peek.pm
lib/Errno.pm
lib/ExtUtils/Install.pm
lib/ExtUtils/Installed.pm
lib/ExtUtils/Packlist.pm
lib/ExtUtils/ParseXS.pm
lib/File/Fetch.pm
lib/File/Path.pm
lib/Filter/Util/Call.pm
lib/Thread.pm
lib/Thread/Queue.pm
lib/Thread/Semaphore.pm
lib/Time/HiRes.pm
lib/threads.pm
lib/threads/shared.pm
```
Since the usage presented is consistent with existing code, I'm going to leave it as it is.  If you want to make a case for your alternative, that should go in a different ticket.

Thank you very much.
Jim Keenan